### PR TITLE
Motd

### PR DESCRIPTION
**Pull request by beatyt**
Took a stab at [motd](https://trello.com/b/4zEJvVmk/jankbot).

As always, feel free to give feedback, etc.

### CODE REVIEW

File: motd/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+LICENSE
+
+Copyright (c) 2014 JankDota
```
Comment (twisterghost) [inline motd/LICENSE:3]: We didn't write this, you can take the copyright ;)

**Comment by twisterghost** [comment]
This looks great, I'll scan through and give pointers, but I'm actually in favor of moving away from having all jankbot modules in this repo, and instead in favor of module authors hosting their own repos and publishing their work to npm using the `jankbot-` prefix. For example, this could be `jankbot-motd`

File: motd/motd.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+// Saves motd.
+exports.save = function() {
+  fs.writeFileSync("./data/motd.json", JSON.stringify(motd));
+}
+
+
+// Handler.
+exports.handle = function(input, source) {
+  input = input.split(' ');
+  userName = friends.nameOf(source);
```
Comment (twisterghost) [inline motd/motd.js:21]: Doesn't look like `userName` is used anywhere here

File: motd/motd.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  if (input[0] == 'motd') {
+	if (friends.isAdmin(source) && input[1] == 'set') {
+	  input.splice(0, 2);
+      var newMotd =  input.join(" ");
```
Comment (twisterghost) [inline motd/motd.js:25]: Indentation is off here a bit

**Comment by twisterghost** [comment]
This is awesome. I'm going to close this PR, though, and suggest that you have it in its own repository under your name to credit you, and publish it to npm yourself. Great work. Let me know either on here or by tagging me somewhere else when you publish it! :+1:

**Comment by beatyt** [comment]
1st time publishing a module to npm but it seems to be there.

**Pull request closed without merging**